framework/src/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1922,7 +1922,7 @@ public final boolean shouldSkipUses(Element element) {
      * @param typeName the fully-qualified name of a type
      * @return true iff the enclosing class of element should be skipped
      */
-    public final boolean shouldSkipUses(String typeName) {
```

Comment:
If we do this, can you add some comment to the javadoc when this method should be overridden.